pandasql/core.py

```diff
@@ -29,10 +29,11 @@ def result_ensured(self, *args, **kwargs):
 
 
 class BaseFrame(object):
-    def __init__(self, name=None, sources=None):
+    def __init__(self, name=None, sources=None, pandas_sources=None):
         self.name = name or _new_name()
         # TODO: deduplicate sources
         self.sources = sources or []
+        self.pandas_sources = pandas_sources or []
```

Comment:
is there a reason we have this as a list? It seems as though it only ever has 1 element. Is there a future use case where we'd add more to it?

Reply:
Hm, can't think of any right now. I did it this way because it's (arguably) cleaner and follows the style of the regular sources. No real preference.